src/file/format/json.rs

```diff
@@ -22,13 +32,15 @@ fn from_json_value(uri: Option<&String>, value: &serde_json::Value) -> Value {
     match *value {
         serde_json::Value::String(ref value) => Value::new(uri, ValueKind::String(value.clone())),
 
-        serde_json::Value::Number(ref value) => if let Some(value) = value.as_i64() {
-            Value::new(uri, ValueKind::Integer(value))
-        } else if let Some(value) = value.as_f64() {
-            Value::new(uri, ValueKind::Float(value))
-        } else {
-            unreachable!();
-        },
+        serde_json::Value::Number(ref value) => {
+            if let Some(value) = value.as_i64() {
+                Value::new(uri, ValueKind::Integer(value))
+            } else if let Some(value) = value.as_f64() {
+                Value::new(uri, ValueKind::Float(value))
+            } else {
+                unreachable!();
+            }
+        }
```

Comment:
This also seems like a formatting change only...

Reply:
Yes, rustfmt did.